### PR TITLE
Always use latest 2 orb

### PR DESCRIPTION
**Pull request by xorima**
This change is [Reviewable](https://reviewable.io/reviews/sous-chefs/ruby_rbenv/256)

### CODE REVIEW

**Review by damacus — DISMISSED** [review]

**Review by damacus — CHANGES_REQUESTED** [review]
Deprecation

File: kitchen.dokken.yml [hunk 1]
```diff
@@ -16,15 +16,15 @@ verifier:
 platforms:
   - name: debian-8
     driver:
-      image: dokken/debian-9
+      image: dokken/debian-8
```
Comment (damacus) [inline kitchen.dokken.yml]: Debian 8 is deprecated

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This Pull Request is probably missing tests.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by xorima** [comment]
libssl-dev required for these servers not libssl-dev1.0

**Comment by xorima** [comment]
@damacus lgtm, separate project to support deb 10 as it just breaks everything currently (no java, some packages just don't exist etc)

**Review by damacus — APPROVED** [review]

**Pull request merged**